lazy-pages/common/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "gear-lazy-pages-common"
```

Comment:
everything from workspace as well

versions for everything of executing and direct correlation to runtime execution must match runtime version